Sources/KituraTemplateEngine/TemplateEngine.swift

```diff
@@ -87,6 +122,16 @@ extension TemplateEngine {
                        options: RenderingOptions, templateName: String) throws -> String {
         return try render(filePath: filePath, context: context, options: options)
     }
+    
+    //Implementation of Codable render for TemplateEngines that do not implement it
```

Comment:
Space after `//`. Also the other comments in this file say `implementation` so please update those too.